cmd/kubeadm/app/cmd/token.go

```diff
@@ -221,7 +220,6 @@ func RunCreateToken(out io.Writer, client clientset.Interface, token string, tok
 		}
 	}
 
-	// TODO: Validate usages here so we don't allow something unsupported
```

Comment:
Can you fix this TODO please?
With this I mean that there should exist a list of the supported usages (signing, authentication), and if you specify something else (like foo) it should error out.

Reply:
actually, I fixed this in another commit of this PR, but @wackxu told me that it's dup of #53929, so I just remove that commit :)

Reply:
then I wouldn't remove the comment here, instead let @wackxu do it in https://github.com/kubernetes/kubernetes/pull/53929

Reply:
OK :)